taca/storage/storage.py

```diff
@@ -2,16 +2,27 @@
 
 import os
 import re
-import shutil
+import csv
 import time
+import shutil
 
+from datetime import datetime
 from multiprocessing import Pool
 
 from taca.log import get_logger
 from taca.utils.config import get_config
 from taca.utils import filesystem, misc
+from statusdb.db import connections as statusdb
 
-def cleanup(days):
+## default thershold days to check on different sites ##
+site_check_days = {'nas' : 2, 'swestore' : 480, 'proc' : 2,
+                   'archive' : 90, 'illumina' : 90, 'analysis' : 90}
+
+def cleanup_nas(days):
+    """
+        Will move the finished runs in NASes and processing server
```

Comment:
Indentation is a bit weird here, check [this](https://www.python.org/dev/peps/pep-0008/#documentation-strings)